### PR TITLE
Materialized views support

### PR DESCRIPTION
**Pull request by kpashka**
Wow, looks like we started this at almost same time :)
https://github.com/sosedoff/pgweb/commit/3af20f932731c78eccd4382d5f05879d75b8341b

If you'll find something useful here - I'm glad to share.

### CODE REVIEW

**Comment by kpashka** [comment]
P.S. Are you using old postgres? Materialized views are removed from `information_schema` since 9.3
http://stackoverflow.com/a/19129980

**Comment by sosedoff** [comment]
Here's the postgres version im using locally:

```
PostgreSQL 9.4.4 on x86_64-apple-darwin14.3.0, compiled by Apple LLVM version 6.1.0 (clang-602.0.53) (based on LLVM 3.6.0svn), 64-bit
```

**Comment by kpashka** [comment]
I am using `9.4.5` on OSX - it just means that materialized views structure should be fetched by different query than the one for tables

**Comment by sosedoff** [comment]
Are you sure about that? If you run `\dm` using `psql` tool, it'll produce the following sql:

```sql
SELECT n.nspname as "Schema",
  c.relname as "Name",
  CASE c.relkind WHEN 'r' THEN 'table' WHEN 'v' THEN 'view' WHEN 'm' THEN 'materialized view' WHEN 'i' THEN 'index' WHEN 'S' THEN 'sequence' WHEN 's' THEN 'special' WHEN 'f' THEN 'foreign table' END as "Type",
  pg_catalog.pg_get_userbyid(c.relowner) as "Owner"
FROM pg_catalog.pg_class c
     LEFT JOIN pg_catalog.pg_namespace n ON n.oid = c.relnamespace
WHERE c.relkind IN ('m','')
      AND n.nspname <> 'pg_catalog'
      AND n.nspname <> 'information_schema'
      AND n.nspname !~ '^pg_toast'
  AND pg_catalog.pg_table_is_visible(c.oid)
ORDER BY 1,2;
```

Thats pretty much what is being used in pgweb and is verified on postgresql 9.3+

File: pkg/statements/sql.go [hunk 1]
```diff
@@ -91,6 +91,21 @@ WHERE
...
+  (case when attnotnull IS TRUE then 'NO' else 'YES' end) as is_nullable,
+  null as character_maximum_length,
+  null as character_set_catalog,
+  null as column_default
+FROM
+  pg_attribute
+WHERE
+  attrelid = $1::regclass AND attnum > 0
+  AND NOT attisdropped;`
+
```
Comment (kpashka) [inline pkg/statements/sql.go]: @sosedoff You're talking about `statements.PG_OBJECTS` query, yep, that works. But I'm talking about next step - `statements.PG_TABLE_SCHEMA`, which is querying `information_schema`, where is no info about materialized views.

Do you have materialized views working now?

Comment (sosedoff) [inline pkg/statements/sql.go]: Ah, i get what you're referring to now, sorry it was not clear to me what the problem was at first.
I'll probably grab a few parts of this PR. Thanks

**Comment by sosedoff** [comment]
Pushed to master

**Pull request closed without merging**